packages/rum-core/src/domain/resource/resourceUtils.ts

```diff
@@ -148,6 +155,7 @@ export function hasValidResourceEntryTimings(entry: RumPerformanceResourceTiming
   // it.
   const areCommonTimingsInOrder = areInOrder(
     entry.startTime,
+    ...(entry.workerStart > 0 ? [entry.workerStart] : []),
```

Comment:
**💬 suggestion:** To makes things simpler, I would not check whether `workerStart` is in order here. In the future I want to get rid of this check altogether. An attempt was already made [here](https://github.com/DataDog/browser-sdk/pull/2688) but was never shipped because we observed too many incoherent results, but I still intend to do something about this.